Source/cielim/Private/ProtobufActor.cpp

```diff
@@ -35,6 +50,9 @@ void AProtobufActor::Tick(float DeltaTime)
     this->UpdateCelestialBodies();
     this->UpdateSpacecraft();
 
+    std::string debugstr = this->vizmessage.DebugString();
+    // UE_LOG(LogTemp, Warning, TEXT("vizmessage: %hs"), debugstr.c_str());
+    GEngine->AddOnScreenDebugMessage(-1, 30.f, FColor::White, FString::Printf(TEXT("vizmessage: %hs"), debugstr.c_str()));
```

Comment:
This must be left over from an older version I had but we can remove these lines of code because they are redundant

Reply:
Removed. Thanks!